harness/determined/cli/user.py

```diff
@@ -445,42 +426,43 @@ def update_token(args: argparse.Namespace) -> None:
         ]),
         cli.Cmd("token", None, "manage access tokens", [
             cli.Cmd("describe", describe_token, "describe token info", [
-                cli.Arg("username", nargs=argparse.ONE_OR_MORE, default=None,
-                        help="name of user to describe token"),
+                cli.Arg("token_id", type=int, nargs=argparse.ONE_OR_MORE, default=None,
+                        help="describe given access token id"),
                 cli.Group(
                     cli.output_format_args["json"],
                     cli.output_format_args["yaml"],
                 ),
             ]),
             cli.Cmd("list ls", list_tokens, "list all active access tokens", [
+                cli.Arg("username", type=str, nargs=argparse.OPTIONAL,
+                        help="list token for the given username", default=None),
                 cli.Arg("--all", "-a", action="store_true", default=None,
                         help="list all access tokens, including revoked & expired tokens"),
+                cli.Arg("--show-active", action="store_true", default=None,
+                        help="list only the active tokens"),
                 cli.Group(
                     cli.output_format_args["json"],
                     cli.output_format_args["yaml"],
                 ),
             ]),
             cli.Cmd("revoke", revoke_token, "revoke token", [
-                cli.Arg("token_id", help="revoke given access token"),
+                cli.Arg("token_id", help="revoke given access token id"),
             ]),
             cli.Cmd("create", create_token, "create token", [
-                cli.Arg("--username", type=str, help="name of user to create token"),
-                cli.Arg("--lifespan", type=str, help="give expiry lifespan"),
-                cli.Arg("--description", type=str, default=None, help="description of new token"),
-                cli.Arg("--file-path", type=str, help="write token to file"),
+                cli.Arg("username", type=str, nargs=argparse.OPTIONAL,
+                        help="name of user to create token", default=None),
+                cli.Arg("--expiration-duration", "-e", type=str,
+                        help="give expiry duration like 2h or 5m or 10s"),
+                cli.Arg("--description", "-d", type=str, default=None,
+                        help="description of new token"),
+                cli.Arg("--output-file", "-o", type=str, help="write token to a file"),
                 cli.Group(
                     cli.output_format_args["json"],
                     cli.output_format_args["yaml"],
                 ),
             ]),
-            cli.Cmd("update", update_token, "update token info", [
```

Comment:
Is there a replacement for this update command or are we making `description` on access tokens immutable here? I'm up for either case, I just want to make sure I understand 😄

Reply:
In our discussion, I didn't find anything on `det token update`, so thought we are not keeping such case. IMO, I would like to keep the `description` on access tokens mutable. I can add it back if we decide to.

Reply:
All good, yea I'm in favor of keeping it; it might be worth modifying the help string to not include "revoke" since we're not exposing revocation through the update command

Reply:
Change the `description` from the positional argument as well.

Reply:
Was this the reason why `det token update` command was ignored?
https://github.com/determined-ai/determined/pull/9867/files#r1776017905

Reply:
Yep! It's likely just swapping the command from update to edit would be the move here 😄

Reply:
Do we want to change it to `det token edit <token_id> --description`?

Reply:
Yep, I think that's more inline with what @azhou-determined had been suggesting since we're currently only supporting description modification. In the future, there might be other mutable fields that we add in that we are thinking about now